### DISCUSSION TITLE
Always visible scroll bar

### DISCUSSION THREAD

**Discussion by hoggatt** [Radiokot/photoprism-android-client#61] [Ideas]
This is an awesome app! Just found it on Fdroid. 

One thing that confused me at first is that the scroll bar is not visible by default. Could this be made a toggleable setting that is on by default?

**Comment by hoggatt** [comment]
One other thing: when I hit the search bar, could a "people" section appear as well?

**Reply by ippocratis** [reply]
> One other thing: when I hit the search bar, could a "people" section appear as well? 

Photoprism android client can use photoprism filters in the search field
Also supports bookmarks

So

If you already have people tagged use the search field to search their names
then
Bookmark the search resault
 
https://docs.photoprism.app/user-guide/search/filters/#using-search-filters

**Reply by hoggatt** [reply]
Yes I'm aware, it would just be a bit more user-friendly this way since people are one of the major reasons people use the search.

**Reply by Radiokot** [reply]
It makes sense, I'll add this feature to the backlog.

**Comment by Radiokot** [comment]
Hi.
It is not visible by default because it is not a true scroll. It doesn't move as you scroll the list, it is rather a control for fast jump to a specific month. This is the way the chronological fast scroll works in the app considering PhotoPrism limitations.
